src/components/grid-aware/PartnersAndSponsorsBlock/PartnersAndSponsorsBlock.tsx

```diff
@@ -2,11 +2,33 @@ import * as React from "react";
 
 import * as s from "./PartnersAndSponsorsBlock.module.css";
 
-type LogoPropType = Readonly<{
+export type LogoPropType = Readonly<{
+  /// The URL of the image
   url: string;
+  /// Alternative text for the image
   alt: string;
+  /// An link for a URL to go to when clicking on the logo
```

Comment:
I just realized that I made a typo here, so I'm going to fix this up right before merging. Thanks for reviewing!

```suggestion
  /// A link for a URL to go to when clicking on the logo
```